src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp

```diff
@@ -0,0 +1,383 @@
+#include <experimental/filesystem>
+#include <fstream>
+
+#include "Client/ComputationToDb/Client.hpp"
+#include "external/Proto/common_types/common_types.pb.h"
+#include "gtest/gtest.h"
+namespace fs = std::experimental::filesystem;
+
+// DBを初期化する
+auto initialize()
+{
+    for (const auto &entry : fs::directory_iterator("/Db/share/"))
+    {
+        if (fs::is_directory(entry.path())) fs::remove_all(entry.path());
+    }
+    for (const auto &entry : fs::directory_iterator("/Db/result/"))
+    {
+        if (fs::is_directory(entry.path())) fs::remove_all(entry.path());
+    }
+}
+
+// shareの取り出し
+// qmpc::ComputationToDbGate::ValueTable Client::readShare(const std::string &data_id);
+TEST(ComputationToDbTest, SuccessReadShareTest)
+{
+    initialize();
+
+    const std::string data_id = "ReadShareTestId";
+    const std::string data =
+        "{\"value\":[[\"1\",\"2\"],[\"3\",\"4\"]]"
+        ",\"meta\":{\"piece_id\":0,\"schema\":[\"attr1\",\"attr2\"]}}";
```

Comment:
Use raw string literal if you don't mind.

-  https://cpprefjp.github.io/lang/cpp11/raw_string_literals.html
- https://en.cppreference.com/w/cpp/language/string_literal

Reply:
@arukuka 
I didn't know that, thanks!
I fixed it in this 0ef7430e1492ac6e03e208db7e468281602b7179.

Reply:
@mdonaka Thank you!  
I checked it.